File: Google Earth Engine/intro.js
var L8: ImageCollection LANDSAT/LC08/C02/T1
var region:Polygon, 4 vertices
var Cor: B4, B3 and B2

var filteredRegion = L8.filterBounds(region);
var filteredMeta = filteredRegion.filterMetadata('CLOUD_COVER', 'less_than', 15)
var filteredDate = filteredMeta.filterDate('2021-01-01', '2023-02-10')
var medianImage = filteredDate.median()
Map.addLayer(medianImage, Cor, "Layer");

File: Google Earth Engine/intro.py
import ee
ee.Initialize()
coordenadas =  "-3.10719,  -60.0261, -3.9000, -64.5000"
x1, y1, x2, y2 = coordenadas.split(",")
geometria = geometry = ee.Geometry.Polygon (
                                            [
                                                [[float(x1), float(y2)],
                                                 [float(x1), float(y1)],
                                                 [float(x2), float(y2)],
                                                 [float(x2), float(y1)]
                                                ]
                                            ]
                                            )
#Queremos as coordenadas centrais da área:
longitude_centro = (float(y2) + float(y1)) / 2
latitude_centro = (float(x2) + float(x1)) / 2
#Definir por data as imagens que queremos:
data = "2017-01-01,2017-12-31"
#Unpacking:
inicio, fim = data.split(",")
print(inicio)
print(fim)
#Filtrar coleções:
#1 - Pegar a coleção desejada, vamos usar o landsat-08
colecao = ee.ImageCollection("LANDSAT/LC08/C02/T1").filterBounds(geometria).filterDate(inicio, fim).filterMetadata('CLOUD_COVER', 'less_than', 100)
#Ver o total de imagens disponíveis:
print(colecao.size().getInfo())
